### ISSUE TITLE
JHipster 5 compatibility

### ISSUE THREAD

**Issue by amarisi** [hipster-labs/generator-jhipster-entity-audit#86]
Hello, I'm wondering if there is any restriction to use this module with the new JHipster 5 version.

I'm getting this whenever I try to set it up on a new project:
```
    ERROR!  I support only JHipster versions greater than >=4.10...
                If you want to use Entity Audit with an older JHipster version, download a previous version that supports the required JHipster version. 
```

Thank you in advance.

**Comment by deepu105** [comment]
Support for version 5 is not coded yet :)

**Comment by ofaklintrafo** [comment]
@deepu105 do you know what is required to get it to pass the tests and become Jhipster 5 compatible

**Comment by deepu105** [comment]
I haven't tried it yet, can you try to generate this on top a Jhipster
project from Jhipster master branch(so that its compatible with JHipster
6). Once you generate see what breaks. I believe it should be easy to fix

Thanks & Regards,
Deepu

**Comment by murdos** [comment]
Issue fixed with #119 ; a new version should be released soon
